Documentation/Contributing/documentation.md

```diff
@@ -35,9 +35,8 @@ When previewing, now you can navigate your browser to [http://127.0.0.1:8000/](h
 
     Please make sure that your Python binary path is included in your `PATH`.
 
-## Running helm-docs
-[helm-docs](https://github.com/norwoodj/helm-docs) is a tool that generates the documentation for a helm chart automatically. If there are changes in the helm chart, you need to run helm-docs manually, and check in the resulting autogenerated md files at the path `/Documentation/Helm-Charts`
+## Making docs
+
+[helm-docs](https://github.com/norwoodj/helm-docs) is a tool that generates the documentation for a helm chart automatically. If there are changes in the helm chart, the developer needs to run  `make docs` (to run helm-docs)  and check in the resulting autogenerated md files under  the path `/Documentation/Helm-Charts`. 
+The CI will check on a PR if helm-docs modifies any checked-in file that the developer forgot to commit and complain in that case. To make it easy to check locally for uncommitted changes generated by helm-docs,   an additional `make` target exists:   just running  `make check-helm-docs`  will run `helm-docs` and complain  if this results in uncommitted generated helm chart doc files. So it is a goog habit to run `make check-helm-docs` locally before creating or updating a PR.   
```

Comment:
Even this line 41 can be removed IMO

Reply:
@travisn  wrote: 

> Even this line 41 can be removed IMO

I didn't remove it completely, but simplified it, removing the "CI" part ...

Reply:
@travisn  wrote:
> Honestly, I don't understand the paragraph. There are some sentence formatting issues and partial sentences.

I updated the PR once more to clarify the paragraph. I hope it is clearer now!

Additonally, I am wondering if we should mention that `make docs` generates more than just helm chart docs